### PR TITLE
Fixes #94 - Add speakers html/css

### LINKED ISSUE #94: HTML/CSS - Speakers - est. 1 day

### PR DESCRIPTION
**Pull request by Ctannerweb**
Add speaker pages to each location

Change name of the team page to staff

Update favicon

### CODE REVIEW

File: 2016/styles/load.css [hunk 1]
```diff
@@ -10,6 +10,7 @@
     --beige10: rgba(251,240,224,.1);
   --black: #171410;
   --brown: #878079;
+  --l-gray: #E3E3E3;
```
Comment (acacheung) [inline 2016/styles/load.css]: lowercase makes it easier to distinguish between letters and numbers

File: 2016/styles/base.css [hunk 2]
```diff
@@ -2,4 +2,5 @@ body {
   font-size: 16px;
   font-family: var(--text);
   color: var(--black);
+  background-color: var(--white)
```
Comment (acacheung) [inline 2016/styles/base.css]: should we add a background color for dark pages?

File: 2016/styles/modules/heading.css [hunk 3]
```diff
@@ -39,6 +39,20 @@
       height: 55px;
     }
   }
+
+  .speakers & {
+    margin-bottom: 24px;
+    margin-left: auto;
+    height: 51px;
```
Comment (acacheung) [inline 2016/styles/modules/heading.css]: can be removed because is in min-width query

File: maine/speakers.html [hunk 4]
```diff
@@ -18,6 +18,208 @@
...
+      <div class="l-speakers">
+        <div class="l-heading">
+          <svg class="heading__icon"><use xlink:href="#flashlight"></use></svg>
+          <h1 class="heading">Speakers</h1>
+        </div>
+        <div class="speaker">
+          <figure class="person">
+            <img class="person__image" src="../2016/images/team/maria-metveeva.jpg" alt="Maria Metveeva">
+            <a class="t-link speaker__social spot-color" href="https://twitter.com/mjacksonw">
+                <svg class="icon-twitter spot-color"><use xlink:href="#twitter"></svg>
```
Comment (acacheung) [inline maine/speakers.html]: too indented

File: maine/speakers.html [hunk 5]
```diff
@@ -18,6 +18,208 @@
...
+            </a>
+            <a class="t-link speaker__social spot-color" href="https://twitter.com/mjacksonw">
+                <svg class="icon-web spot-color"><use xlink:href="#web"></svg>
```
Comment (acacheung) [inline maine/speakers.html]: too indented

File: maine/speakers.html [hunk 6]
```diff
@@ -18,6 +18,208 @@
...
+            </a>
+          </figure>
+          <div class="speaker__desc">
+            <h3 class="speaker__name">Jackson Wilkinson</h3>
+            <h4 class="speaker__position">Product Design &amp; Engineering Leader at <a class="t-link spot-color" href="">Google</a></h4>
+            <p class="speaker__bio">Jackson is a product design and engineering leader and the Founder of Kinsights. Previously, Jackson was Director of User Experience at Posterous (acquired by Twitter), and design lead at LinkedIn. He also founded the User Experience group at Viget, a top interactive agency based in Washington, DC.</p>
+            <p class="speaker__bio">Jackson is a product design and engineering leader and the Founder of Kinsights. Previously, Jackson was Director of User Experience at Posterous (acquired by Twitter), and design lead at LinkedIn. He also founded the User Experience group at Viget, a top interactive agency based in Washington, DC.</p>
+            <p class="speaker__bio">Friday, 7:00-8:00 pm: <a class="t-link spot-color" href="#">Stop Guessing Colors</a></p>
```
Comment (acacheung) [inline maine/speakers.html]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

File: maine/speakers.html [hunk 7]
```diff
@@ -18,6 +18,208 @@
     </script>
     <script type="text/javascript" src="../js/2015/scripts.js"></script>
```
Comment (acacheung) [inline maine/speakers.html]: 2016

File: 2016/styles/base.css (hunk 2 above)
Comment (Ctannerweb) [inline 2016/styles/base.css]: Yep :+1: will add black background color for dark pages

File: 2016/styles/base.css [hunk 8]
```diff
@@ -2,4 +2,10 @@ body {
   font-size: 16px;
   font-family: var(--text);
   color: var(--black);
+  &:not(.is-dark) {
+    background-color: var(--white)
```
Comment (acacheung) [inline 2016/styles/base.css]: missing semicolon

File: 2016/styles/base.css [hunk 9]
```diff
@@ -2,4 +2,10 @@ body {
...
+  }
+  &.is-dark {
+    background-color: var(--black)
```
Comment (acacheung) [inline 2016/styles/base.css]: missing semicolon

Comment (acacheung) [inline 2016/styles/base.css]: should do this in the same way that text color is being done in theme.css

File: maine/speakers.html [hunk 10]
```diff
@@ -16,8 +16,197 @@
...
+        </symbol>
+        <symbol id="maine" viewBox="0 0 2627 1024">
```
Comment (acacheung) [inline maine/speakers.html]: update maine svg to match the one in performance

        <symbol id="maine" viewBox="0 0 3222 1024">
          <path d="M3094.332 525.303c-269.032 0-611.128 332.576-689.588 332.576 0-83.841 32.942-164.782 78.614-231.527 31.874 16.724 80.671 41.123 94.067 41.123 80.172 0 214.903-123.783 214.903-186.282 0-17.663-49.641-50.319-49.641-50.319s-59.795-57.614-88.328-57.614c-81.184 0-242.704 118.223-308.061 274.986-44.879 73.401-151.473 192.261-177.309 192.261 0-133.968 147.362-291.581 178.007-336.567 13.475-19.78-56.84-64.319-71.799-64.319-103.216 0-339.283 297.038-356.111 320.224 32.161-112.189 146.689-226.735 146.689-247.677 0-19.446-61.33-59.834-83.768-59.834-15.937 0-102.805 122.244-151.807 219.347-47.088 71.459-87.085 135.062-143.624 156.86 0-121.835 178.754-330.585 178.754-359.753 0-26.999-74.601-71.801-86.012-71.801-17.681 0-96.566 128.18-145.825 235.266-50.445 81.5-146.523 195.216-174.945 195.216 0-84.196 143.456-280.897 143.456-301.84 0-26.426-55.789-63.822-77.125-70.070-17.557-23.17-64.177-61.564-88.859-61.564-111.133 0-373.945 252.166-373.945 375.459 0 57.012 91.979 120.196 117.906 120.196 45.622 0 212.908-205.459 231.621-232.384-13.977 23.933-35.699 73.581-35.699 118.601 0 39.209 53.301 98.001 113.883 98.001 14.896 0 64.76-61.050 103.531-112.565 4.559 60.467 76.001 115.104 120.876 115.104 13.122 0 50.463-44.314 83.951-91.761 11.068 56.302 75.857 85.030 91.868 85.030 11.164 0 106.251-156.032 244.574-277.478-28.42 40.387-55.346 102.215-55.346 180 0 82.518 76.286 123.655 109.195 123.655 19.261 0 91.9-86.139 130.415-137.544-0.653 8.212-0.994 16.472-0.994 24.773 0 47.032 79.781 144.361 163.113 144.361 57.065 0 460.060-364.735 659.818-364.735 57.065 0 60.023 24.95 66.577 24.95 6.552 0 14.671-13.724 14.671-24.593s-51.617-37.761-127.704-37.761zM2704.227 459.948c0 36.684-122.665 138.558-168.869 138.558-7.68 0-18.122-1.832-28.3-4.162 64.803-81.060 145.783-134.396 197.169-134.396zM1142.193 840.188c0-66.188 234.743-348.214 358.899-354.197-53.851 93.491-323.852 354.197-358.899 354.197z"></path>
          <path d="M994.185 986.396c-8.165 0-15.146 6.939-25.367 6.939-8.292 0-18.337-30.915-18.337-57.227 0-219.574 314.436-631.116 314.436-642.584-0.497-36.581-81.978-82.867-98.744-82.639s-298.329 336.647-368.844 442.906c112.626-283.425 256.503-430.322 256.503-452.79 0-18.292-78.259-69.722-91.978-69.722-24.974 0-222.417 209.917-398.545 454.476 101.601-245.611 257.169-449.476 257.169-474.477 0-42.632-114.232-111.575-136.046-111.278-39.63 0.539-543.836 122.553-609.385 122.553-45.377 0-56.375-16.925-60.076-16.925-8.16 0-15.136 3.179-14.97 15.373 0.167 12.196 86.154 101.432 124.263 101.432 113.425 0 447.348-89.644 556.546-116.57 0 27.434-344.989 521.574-344.989 736.888 0 37.489 74.171 124.334 89.877 124.334s67.188-242.506 377.624-601.821c-67.31 98.721-164.591 346.511-164.591 415.409 0 21.341 64.566 83.020 91.995 83.020 9.219 0 59.192-142.672 287.672-411.436-55.815 82.348-170.358 287.28-170.358 428.669 0 89.721 77.84 143.073 112.050 143.073 24.789 0 42.133-14.46 42.133-23.435 0.001-6.098-3.874-14.169-8.039-14.169z"></path>
          <path d="M1917.393 377.47c7.752-0.016 88.641-83.001 88.568-119.172-0.057-28.42-58.433-63.622-89.264-63.622-15.521 0-80.097 77.404-80.044 103.691 0.109 54.256 66.53 79.131 80.739 79.103z"></path>
        </symbol>

**Pull request merged**